packages/dvilela/skills/memeooorr_abci/handlers.py

```diff
@@ -142,6 +143,23 @@ def load_fsm_spec() -> Dict:
         return yaml.safe_load(spec_file)
 
 
+def get_password_from_args() -> Optional[str]:
+    """Extract password from command line arguments."""
+    args = sys.argv
+    try:
+        password_index = args.index("--password")
+        if password_index + 1 < len(args):
+            return args[password_index + 1]
+    except ValueError:
+        pass
+
+    for arg in args:
+        if arg.startswith("--password="):
+            return arg.split("=", 1)[1]
+
+    return None
```

Comment:
I always disliked this way, looks a little hacky. I prefer this instead. Wdyt?

```suggestion
    arg_parser = ArgumentParser()
    arg_parser.add_argument(
        "--password",
        type=str,
        help="Password to decrypt the Ethereum private key.",
    )
    args = arg_parser.parse_args()
    return args.password
```

Reply:
Great suggestion. 
Tweaked a bit to deal with unkown arguments like -s: https://github.com/dvilelaf/meme-ooorr/pull/297/commits/50dcffe504349e56b2cb558f4bc6e6360432e7e7

Also two more improvements/fixes to related code: https://github.com/dvilelaf/meme-ooorr/pull/297/commits/4987ca8ba72ccbcd1a306e1b92019d6a3f7c45e3 https://github.com/dvilelaf/meme-ooorr/pull/297/commits/d6616ecf48accb4bdb4518ab80c53d83c111edce